packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -131,7 +131,6 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
   }, [adapter]);
 
   const messageThreadProps = usePropsFor(MessageThread);
```

Comment:
Could you add an early loading for the rich text send box? Please see 
`  /* @conditional-compile-remove(rich-text-editor) */
  useEffect(() => {
    // if rich text editor is enabled, the rich text editor component should be loaded early for good UX
    if (richTextEditor !== undefined && richTextEditor) {
      // this line is needed to load the Rooster JS dependencies early in the lifecycle
      // when the rich text editor is enabled
      loadChatMessageComponentAsRichTextEditBox();
    }
  }, [richTextEditor]);`
for the refference